src/Esri.ArcGISRuntime.Toolkit/Shared/UI/Controls/ScaleLine/ScaleLine.cs

```diff
@@ -32,23 +37,34 @@ namespace Esri.ArcGISRuntime.Toolkit.UI.Controls
     /// The ScaleLine Control generates a line representing
     /// a certain distance on the map in both Metric and US units.
     /// </summary>
-    public class ScaleLine : Control
+    public partial class ScaleLine : Control
     {
         private TextBlock _usValue;
         private TextBlock _usUnit;
         private TextBlock _metricValue;
         private TextBlock _metricUnit;
         private Rectangle _metricScaleLine;
         private Rectangle _usScaleLine;
+        private static double _dpi =
+#if __IOS__
+            96 * UIKit.UIScreen.MainScreen.Scale;
+#else
+            96;
```

Comment:
Always 96. The value is calculated based on a dips value already, so this doesn't change with screenscale. Probably just revert all _dpi changes back to the fixed value.